.gitlab/e2e_install_packages/common.yml

```diff
@@ -23,20 +17,6 @@
     TEAM: agent-delivery
     EXTRA_PARAMS: --osversion $E2E_OSVERS --platform $E2E_PLATFORM --cws-supported-osversion $E2E_CWS_SUPPORTED_OSVERS --major-version $AGENT_MAJOR_VERSION --arch $E2E_ARCH --flavor $FLAVOR
 
-.new-e2e_script_upgrade6:
-  stage: e2e_install_packages
-  variables:
-    TARGETS: ./tests/agent-platform/upgrade
-    TEAM: agent-delivery
-    EXTRA_PARAMS: --osversion $E2E_OSVERS --platform $E2E_PLATFORM --arch $E2E_ARCH --flavor $FLAVOR
-  parallel:
-    matrix:
-      - START_MAJOR_VERSION: [5, 6]
```

Comment:
do we run any agent 5 test here? 🤔

Reply:
There's an "upgrade 5" test I came across somewhere, I'm not familiar with it (and kept focused on the 6's here) but I assumed it was upgrading 5->7? If that's the case we may also want to keep an upgrade 6->7 test?